src/ctx.cpp

```diff
@@ -133,7 +133,7 @@ zmq::ctx_t::~ctx_t ()
 
     //  If we've done any Curve encryption, we may have a file handle
     //  to /dev/urandom open that needs to be cleaned up.
-#ifdef ZMQ_HAVE_CURVE
+#if defined (ZMQ_USE_TWEETNACL)
```

Comment:
Ah... I fixed this before I fix the CMAKE variable then. So, this error was cause by the same problem. I will change this back. Thanks.